CONTRIBUTING.md

```diff
@@ -196,62 +196,64 @@ You can set the `DEBUG` environment variable to turn on debugging logs. E.g. `DE
+To get around this, we inline some of these dependencies' types in `packages/vite/types`. This way, we can still expose the typing but bundle the dependency's source code.
 
-Use `pnpm run check-dist-types` to check bundled types does not rely on types in `devDependencies`. If you are adding `dependencies`, make sure to configure `tsconfig.check.json`.
+Use `pnpm run check-dist-types` to check that the bundled types do not rely on types in `devDependencies`. If you are adding `dependencies`, make sure to configure `tsconfig.check.json`.
 
-### Think before adding yet another option
+### Think Before Adding Yet Another Option
 
-We already have many config options, and we should avoid fixing an issue by adding yet another one. Before adding an option, try to think about:
+We already have many config options, and we should avoid fixing an issue by adding yet another one. Before adding an option, consider whether the problem:
 
-- Whether the problem is really worth addressing
-- Whether the problem can be fixed with a smarter default
-- Whether the problem has workaround using existing options
-- Whether the problem can be addressed with a plugin instead
+- is really worth addressing
+- can be fixed with a smarter default
+- has workaround using existing options
+- can be addressed with a plugin instead
 
-## Docs translation contribution
+## Docs Translation Contribution
 
-If you would like to start a translation in your language, you are welcome to contribute! Please join [the #translations channel in Vite Land](https://chat.vitejs.dev) to discuss and coordinate with others.
+If you would like to start a translation in your language, you are welcome to contribute! Please join the [#docs channel](https://discord.com/channels/804011606160703521/855049073157341234) in [Vite Land](https://chat.vitejs.dev) to discuss and coordinate with others.
 
-The english docs are embedded in the main Vite repo, to allow contributors to work on docs, tests and implementation in the same PR. Translations are done by forking the main repo.
+The [English docs are embedded in the main Vite repo](https://github.com/vitejs/vite/tree/main/docs) to enable contributors to work on docs, tests, and implementation in the same PR. Translations of those docs are contained in [separate repos in the `vitejs` org](https://github.com/vitejs?q=docs&type=all&language=&sort=).
 
-### How to start a translation repo
+### How To Start a Translation Repo
 
-1. In order to get all doc files, you first need to clone this repo in your personal account.
-2. Keep all the files in `docs/` and remove everything else.
+1. Create a new Git repo, named `docs-LANGUAGE_CODE`, where `LANGUAGE_CODE` is the [ISO 639-1 code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) for the target language (e.g. `docs-fr` for French translations):
```

Comment:
I wonder if we should create a template repo for translation docs that users could easily fork. That would avoid this whole section of manual steps.

Reply:
I created a [template for Vite docs](https://github.com/tony19/vite-docs-template).

Reply:
Interesting! That looks a lot handy than the current explanation. I'm not familiar with docs development in general. But maybe @ShenQingchuan or @mrcego have some thoughts on this.

Reply:
Quite good! I found that the template project has already setup a ryucho Github Action.

Reply:
I'm agree, the new template will be useful and a good start for incoming translations. Nice the ryu-cho config.

Reply:
@ShenQingchuan @mrcego Thanks! I based the template off of your repos in `doc-es` and `docs-cn`.